.github/workflows/main.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: ▶️ Run Typecheck
         run: npm run typecheck
 
+      - name: ▶️ Run Vitest
+        run: npm run test:unit
+
```

Comment:
I thought about collapsing the the `▶️ Run *` steps into one `npm test` but thought it would be a better fit another time.